### DISCUSSION TITLE
`--color-scale` introduces breaking changes?

### DISCUSSION THREAD

**Discussion by kohane27** [eza-community/eza#649] [Q&A, Howto, Help]
Hello there. Hope you're doing well. Thank you for forking and maintaining this awesome awesome project.

This morning I realized the following stopped working:

```
✖ eza -G --across -x -F --icons --all --group-directories-first --color always --color-scale
eza: Flag --color-scale needs a value (choices: all, size, age)
```

Uisng `eza --help`:

```
--colo[u]r-scale           highlight levels of 'field' distinctly(all, age, size)
```

and `eza -G --across -x -F --icons --all --group-directories-first --color always --color-scale=all` works.

It really isn't an issue, but I was wondering if there's a breaking changes list for reference, and if there's a better way to inform users of any BR.

Thank youa

**Comment by cafkafk** [comment]
We accidentally introduced a breaking change, which wasn't intentional (normally we do label breaking changes in the release notes). It's already been fixed in the meantime thou. See https://github.com/eza-community/eza/issues/634 for more info.
